tests/resources/Util.robot

```diff
@@ -105,8 +105,9 @@ Cleanup VIC Appliance On Test Server
 
 Run VIC Machine Delete Command
     [Tags]  secret
-    ${output}=  Run  bin/vic-machine-linux delete --name=${vch-name} --target=%{TEST_URL} --user=%{TEST_USERNAME} --password=%{TEST_PASSWORD} --force=true --compute-resource=%{TEST_RESOURCE} --timeout %{TEST_TIMEOUT}
-    Run Keyword And Ignore Error  Should Contain  ${output}  Completed successfully
+    ${rc}  ${output}=  Run And Return Rc And Output  bin/vic-machine-linux delete --name=${vch-name} --target=%{TEST_URL} --user=%{TEST_USERNAME} --password=%{TEST_PASSWORD} --force=true --compute-resource=%{TEST_RESOURCE} --timeout %{TEST_TIMEOUT}
+    Should Be Equal As Integers  ${rc}  0
+    Should Contain  ${output}  Completed successfully
```

Comment:
I also added the following as a start to this before I got distracted this past several weeks with nimbus business:

    ${out}=  Run  govc ls vm
    Log  ${out}
    ${out}=  Run  govc datastore.ls
    Log  ${out}
    ${out}=  Run  govc ls host/*/Resources/*
    Log  ${out}

I think we really need to also add a final check to make sure everything got cleaned up something to the effect of:
    ${out}=  Run  govc datastore.ls
    Should not contain  ${out}  ${vch-name}

And do that for vms and pools as well